util/filesystem.go

```diff
@@ -18,6 +19,15 @@ import (
 	"github.com/mackerelio/mackerel-agent/logging"
 )
 
+type DfStat struct {
+	Name      string
+	Blocks    uint64
```

Comment:
[suggestion]
"Blocks" and "Capacity" are named "size" and "percent_used" in spec. Those mismatch are confusing, so how about using same names as spec?